### ISSUE TITLE
Dashboard filter for time series data

### ISSUE THREAD

**Issue by marketingnerds** [metabase/metabase#2820]
I love the new filtering capability in v.18.0 for tabular data where I'm including category information in the query output.

It would be a huge help to me (and others, I hope) to be able to essentially put a where clause on a dashboard to control time series data as well. For example, I have 18 dashboards, one for each of 18 locations, which all show the same time series data. That means that I also have 18 sets of 6 cards, which are in turn composed of 1 to 4 (stacked) questions. You get the idea. A where clause would get me down to one dashboard.

Summary: I'd like to use a dashboard filter on a category for time series data.

**Comment by salsakran** [comment]
You can use a category as a filter as well. See below - 


![screen shot 2016-06-23 at 11 44 43 am](https://cloud.githubusercontent.com/assets/107439/16315537/8a79a756-3937-11e6-8f6f-9367e5ac6fee.png)

**Comment by mazameli** [comment]
What Sameer said, but also, as the resident touchy-feely UX guy, I'd love to know what led you to the perception that this wasn't possible, or what was confusing about setting these filters up.

**Comment by marketingnerds** [comment]
Thanks for responding. I think the issue is that I pretty much only use SQL queries due to a need for table joins and/or sub-selects. I have the appropriate fields marked as "Category" in the Data Model section of Admin but that doesn't seem to carry over for SQL queries. I was successful in testing the category feature using a simple time series card.

Any best practice recommendations? Should I be creating views to eliminate joins, then building my cards using the "friendly" interface options?

**Comment by salsakran** [comment]
Gotcha. SQL parameters are coming! #2348 

In the meantime, the main pattern we've found useful is to use SQL Views to create relatively "clean tables" that can be used without joins. We've found that for most end user facing analytics, it's generally worth taking the time you would spend writing SQL and instead create 5-10 "analytics" tables for end users, hide the rest and have those be the workhorses. 

We'll be adding machinery to do that directly in Metabase, eg #2144 with time, but that's the pattern we're recommending and using internally.
